website/docs/r/application_gateway.html.markdown

```diff
@@ -191,6 +191,8 @@ A `backend_http_settings` block supports the following:
 
 * `request_timeout` - (Required) The request timeout in seconds, which must be between 1 and 86400 seconds.
 
+* `pick_host_name_from_backend_address` - (Optional) Whether host header should be picked from the host name of the backend server.

```

Comment:
Could we show the default value to optional attributes?
```suggestion
* `pick_host_name_from_backend_address` - (Optional) Whether host header should be picked from the host name of the backend server. Defaults to `false`.
```

Reply:
done